pkg/controller/directimagemigration/task.go

```diff
@@ -277,3 +280,25 @@ func (t *Task) getDestinationClient() (compat.Client, error) {
 	}
 	return client, nil
 }
+
+// Get the extended phase description for a phase.
+func (t *Task) getPhaseDescription(phaseName string) string {
+	// Log the extended description of current phase
+	if phaseDescription, found := PhaseDescriptions[t.Phase]; found {
+		return phaseDescription
+	}
+	t.Log.V(4).Info("Missing phase description for phase: " + phaseName)
```

Comment:
I almost want to suggest this should be an error. 

The only time I can see this happening is if 
1. unknown phase is set in the status, and this will error somewhere else as well right?
2. someone added a phase but did not add a description

for the 2nd use case making this loud by default might make some sense, as the first use case is already loud just somewhere else? WDYT?

Reply:
+1, My intent was to help resolve the thing you mention in `2. someone added a phase but did not add a description
`.  I found a few phases missing descriptions and wanted the controller to report that so we can fix it as you mentioned.

I don't believe there is any way for us to end up in an undefined phase (your concern in `1.`) unless someone manually sets the phase. This would be bad obviously, I don't think it's likely though given that Phase is just advanced by `t.next()` according to itinerary contents.

I'm torn on how loud to make this. My first instinct was to make it an error as well, but then I became concerned that if I don't manage to run migrations that hit every possible phase before the next z-stream release, we'll end up possibly shipping a z-stream that shoots out error messages (with stack traces etc) on every migration.

Reply:
I think I am ok with the compromise and a follow-up item to come back and make them errors in the future?

Reply:
Sounds good. I think once we shore up the gaps in missing labels it would make sense to bump this up to error.